src/jit/morph.cpp

```diff
@@ -16910,13 +16910,15 @@ void Compiler::fgMorph()
     fgDebugCheckBBlist(false, false);
 #endif // DEBUG
 
+#ifndef _TARGET_ARM_
```

Comment:
It would be good to have a comment here about why this is not done on ARM.
Is this a legacy-only issue?